### PR TITLE
PhpUnit versions

### LINKED ISSUE #8: Add in support for the latest version of PHPUnit
As @paulgibbs noted in: https://github.com/Chassis/Chassis/pull/214#issuecomment-235851016 we are currently installing an old version of PhpUnit. This was be design as the WordPress core tests were failing with the latest version of PHPUnit.

We should consider adding in the latest version and making the version configurable.

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #8.

### CODE REVIEW

File: chassis.pp [hunk 1]
```diff
@@ -1,2 +1,3 @@
+$tester_config = sz_load_config()
```
Comment (rmccue) [inline chassis.pp]: This should be passed into the resource, rather than used as a global, I think

File: modules/tester/manifests/init.pp [hunk 2]
```diff
@@ -6,15 +6,23 @@
 		ensure => directory,
 	}
 
+	if versioncmp( "$tester_config[php]", '7.0' ) >= 0 {
+		$phpunit_package_name = "phpunit.phar"
+		$phpunit_repo_url     = "https://phar.phpunit.de/phpunit.phar"
+	} else {
+		$phpunit_package_name = "phpunit-old.phar"
```
Comment (rmccue) [inline modules/tester/manifests/init.pp]: We can just use `phpunit.phar` for both of these, should simplify the other changes.

File: chassis.pp (hunk 1 above)
Comment (BronsonQuick) [inline chassis.pp]: Good idea! Done!

File: modules/tester/manifests/init.pp (hunk 2 above)
Comment (BronsonQuick) [inline modules/tester/manifests/init.pp]: I did some more testing around this. `5.6+`can use `phpunit.phar` but the older versions need `phpunit-old.phar` so I've adjusted the logic accordingly. Thanks for the review!

**Pull request merged**